cluster/sbin/snappy-nodes.sh

```diff
@@ -338,12 +338,41 @@ function getNumLeadsOnHost() {
   echo $numLeadsOnHost
 }
 
+function scpConf() {
+ var=$(ip addr | grep 'state UP' -A2 | tail -n1 | awk '{print $2}' | cut -f1  -d'/')
```

Comment:
Please use a meaningful variable name so that we can get the idea of what is being tried to be taken out from the conf.

Reply:
Resolved